### PR TITLE
HPCC-16980 Remove tabs accidently introduced in previous fix

### PR DESCRIPTION
**Pull request by rpastrana**
- Removes all tabs

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@Michael-Gardner please review.

File: system/jlib/jsmartsock.cpp [hunk 1]
```diff
@@ -430,25 +430,25 @@ void CSmartSocketFactory::setStatus(SocketEndpoint &ep, bool status)
...
-			SocketEndpoint ep = sep->ep;
-			ep.getUrlStr(url);
-		}
-	}
-	return url;
+    SmartSocketEndpoint * sep = nextSmartEndpoint();
+    if (sep)
+    {
+        SocketEndpoint ep;
+        if(useHostName && sep->name.length())
```
Comment (Michael-Gardner) [inline system/jlib/jsmartsock.cpp]: Space between the `if (` to be consistent?

Comment (rpastrana) [inline system/jlib/jsmartsock.cpp]: the file itself has instances of both formats, not worth changing.

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**